testsuites/CBLTester/System_test_multiple_device/test_p2p_custom_conflict_resolution.py

```diff
@@ -0,0 +1,487 @@
+import pytest
+
```

Comment:
Can you please follow same comments written for SGW and CBL replication

Reply:
done